.travis.yml

```diff
@@ -3,12 +3,11 @@ jdk:
   - oraclejdk8
 
 install:
-  # Install Srclib
-  - mkdir -p $HOME/bin
-  - export PATH=$PATH:$HOME/bin
-  - curl -Lo $HOME/bin/srclib.gz https://srclib-release.s3.amazonaws.com/srclib/0.1.1-no-docker5/linux-amd64/srclib.gz 
-  - gunzip -c $HOME/bin/srclib.gz > $HOME/bin/srclib
-  - sudo chmod +x $HOME/bin/srclib
+# installing srclib
+  - mkdir $HOME/gocode
```

Comment:
As far as I know, creating the GOPATH directory isn't needed. When you `go get`, it will be created if it doesn't already exist. Just FYI.